src/test/java/org/jvnet/hudson/plugins/nextbuildnumber/JobTestBase.java

```diff
@@ -21,9 +21,10 @@ public void setUp() {
 
     @SuppressWarnings("rawtypes")
     protected void performNextBuildNumberChange(Job project, String currenNumber, String changeToNumber) throws Exception {
-        HtmlForm form = jenkins.createWebClient().getPage(project, "nextbuildnumber").getFormByName("nextbuildnumber");
+        HtmlForm form;
+        form = jenkins.createWebClient().getPage(project, "nextbuildnumber").getFormByName("nextbuildnumber");
```

Comment:
Why split this into two lines?

Reply:
I can't remember why I started that, sorry.
Back to a one-liner.